src/borg/archiver.py

```diff
@@ -4119,8 +4215,8 @@ def define_borg_mount(parser):
                                help='only consider numeric user and group identifiers')
         subparser.add_argument('--same-chunker-params', dest='same_chunker_params', action='store_true',
                                help='Override check of chunker parameters.')
-        subparser.add_argument('--sort', dest='sort', action='store_true',
-                               help='Sort the output lines by file path.')
+        subparser.add_argument('--sort', dest='sort', action='append', nargs='?', const='path', metavar='FIELD',
+                               help='Sort the output lines by one or more fields. Example: --sort=">size_added" --sort=path. Use multiple --sort options for tie-breakers. Default (when omitted or without value) is path ascending.')
```

Comment:
Question: What's the current behaviour without `--sort`?

Even though it likely impacts nothing, I'd rather keep it unsorted without `--sort`. However, like with unknown fields above, really just an IMHO, no strong opinion about this.